tests/net/lib/lwm2m/lwm2m_registry/src/lwm2m_registry.c

```diff
@@ -352,6 +385,158 @@ ZTEST(lwm2m_registry, test_strings)
 	zassert_equal(ret, -ENOMEM);
 }
 
+#if CONFIG_TEST_DEPRECATED
+/* Don't test deprecated functions on Twister builds */
```

Comment:
Not sure I follow - what's the point of adding this test then if we don't want to run it with the CI?

Reply:
It allows us to test those functions manually. I don't think they will change much, so I don't think it is relevant to keep running those in CI.

But I just wrote the tests once to verify and check for code coverage. I have no strong opinion whether I should drop those, or keep for manual testing purposes.

Reply:
If it's already implemented and you plan to use it for the time being I think it could stay - just please remember to remove the test cases when we decide to remove the deprecated APIs.